### PR TITLE
[#135664147] Network Account Info - Social Zombie

### PR DESCRIPTION
**Pull request by jonajonlee**
[..see related ticket from social-zombie..](https://github.com/influentialpublishers/social-zombie/pull/6)

## Deliverable
https://www.pivotaltracker.com/story/show/135664147

- Add getAccountByUsername function to the SocialZombie object
- Add unit tests for lib/socialzombie
- Added UnsupportedSocialZombieNetwork Error

### CODE REVIEW

File: lib/error.js [hunk 1]
```diff
@@ -38,7 +41,7 @@ TooManyRecords.throw = R.curry((store, table, field, constraint, count) => {
 class NotFound extends CussError {
```
Comment (jxbadam) [inline lib/error.js:41]: Shouldn't this be KussError ?

File: lib/socialzombie/index.js [hunk 2]
```diff
@@ -10,18 +11,21 @@ const NETWORKS = [
...
+  if (R.not(isValidNetwork(network)))
+    Err.UnsupportedSocialZombieNetwork.throw(network)
+
+  return client.get(`/user/${network}/username/${username}`)
+
 }
 
 
-// getTimelinePage :: Client -> String, String, String
+// getTimelinePage :: Client -> String -> String -> String
```
Comment (jxbadam) [inline lib/socialzombie/index.js:24]: Doesn't this imply a currying that you aren't doing?

**Review by jxbadam — CHANGES_REQUESTED** [review]

**Comment by scull7** [comment]
Social zombie and kuss should not be linked in any way. 😱

**Comment by jonajonlee** [comment]
@scull7 moved social zombie to social-hoarder. [see new pr](https://github.com/influentialpublishers/social-hoarder/pull/81)

**Pull request closed without merging**